README.md

```diff
@@ -1,78 +1,270 @@
+git clone https://github.com/joticajulian/koinos-contracts-as
 
-## Verify code
+```
+ 
+
+Change into the directory of the SDK by using the following command:
 
-In order to verify the authenticity of the deployed code, this repo contains a Dockerfile to compile the code and get the wasm file with its corresponding sha256 identifier. Then these values can be compared with the code deployed in the blockchain to prove its authenticity.
+ ```
 
-Run:
+cd koinos-contract-as
 
 ```
-yarn token:docker
+
+  
+
+Install the dependencies within the folder with yarn by using the following command:
+ 
+
 ```
 
-You should see at the end some info from the contract selected:
+yarn install
 
 ```
-contract: token
-file:     /contracts/contracts/token/build/release/token.wasm
-size:     35743 bytes (34.91 kB)
-sha256:   b431afde0d72b4398584784c8142de9d7f33452294917af6c402cc0f70f9b85f
+
+  
+
+## Step 2: Modify your .env for the deployer
+
+  
+
+The `koinos-contract-as` repo contains a deployment script so you do not need manually deploy via the `koinos-cli` wallet. To use the automatic deployer, you must set it up. To do this clone the `env.example` file and modify it. To begin, clone and rename the file as `.env` using the following command:
+
+
+```
+
+cp env.example .env
+
+```
+
+Open `.env` with the editor of your choice. Once open, it should appear something like this:
+
+
 ```
 
-## Acknowledgments
+HARBINGER_MANA_SHARER_PRIVATE_KEY=
+HARBINGER_TOKEN_CONTRACT_PRIVATE_KEY=
+HARBINGER_TOKEN_CONTRACT_ID=  
+
+MAINNET_MANA_SHARER_PRIVATE_KEY=
+MAINNET_TOKEN_CONTRACT_PRIVATE_KEY=
+MAINNET_TOKEN_CONTRACT_ID=
+
+```
+
+  
+If you are developing on test net, fill in the information for `HARBINGER`, if you are developing on main net, fill in the information for `MAINNET`.
+
+  
+
+- The `_MANA_SHARER_PRIVATE_KEY` is the private key to a Koinos wallet that containts liquid $KOIN. This wallet is used to cover the Mana cost of uploading the `.wasm` and `.abi` file to the Koinos Blockchain. If you are working on testnet and need testnet $KOIN ($tKOIN) then head over the discord faucet to get some for free. For main net, you will need to buy $KOIN from a CEX or DEX.
+
+ 
+- The `_TOKEN_CONTRACT_PRIVATE_KEY` is the private key to the wallet address that will hold your smart contract. You may create a wallet using `koinos-cli` or any of the 3rd party wallets.
+
+  
+
+- The `_TOKEN_CONTRACT_PRIVATE_KEY` is the wallet address associated with the `_TOKEN_CONTRACT_PRIVATE_KEY`.
+
+  
+
+In total, you will need two wallets and their private keys. One wallet contains liquid $KOIN (or $tKOIN), while the other can have no $KOIN. The purpose of this is so you can deploy a smart contract into an empty wallet and not end up with any dust in it.
+
+  
+
+## Step 3: Modify Token.ts
+
+  
+
+Enter the `contracts` folder and you will find two folders.
+
+  
+
+-  `manasharer` is a Mana Sharing contract that is used to deploy the token contract.
```

Comment:
the `manasharer` contract is not used to deploy the token contract. This is separate contract, not related to `token`.